### PR TITLE
Adds route from guest cluster subnet to vpc peering connection for host cluster cidr

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/1799

We need a route in the guest cluster public subnet that points to the vpc peering connection for the host clusters vpc cidr block. This adds that route.

### CODE REVIEW

File: resources/aws/subnet.go [hunk 1]
```diff
@@ -169,3 +169,25 @@ func (s *Subnet) MakePublic(routeTable *RouteTable) error {
...
+	}
+
+	vpcPeeringConnectionID, err := vpcPeeringConnection.GetID()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if _, err := s.Clients.EC2.CreateRoute(&ec2.CreateRouteInput{
+		RouteTableId:           &routeTableID,
+		DestinationCidrBlock:   &vpc.CidrBlock,
```
Comment (JosephSalisbury) [inline resources/aws/subnet.go]: This is incorrect. The destination cidr is the cidr of the host vpc.

**Comment by JosephSalisbury** [comment]
screen shot 2017-09-01 at 11 24 03

Looks good on gauss

**Comment by MarcelMue** [comment]
I also tested this locally with `gauss`, seems to work fine. Unsure if this could break anywhere, looking for some reasons and getting this merged.

**Comment by xh3b4sd** [comment]
You need to merge master.

File: resources/aws/subnet.go [hunk 2]
```diff
@@ -176,3 +176,25 @@ func (s *Subnet) MakePublic(routeTable *RouteTable) error {
 
 	return nil
 }
+
+func (s *Subnet) AddHostVPCRoute(routeTable *RouteTable, vpcPeeringConnection *VPCPeeringConnection) error {
+	routeTableID, err := routeTable.GetID()
```
[the remark this answers is not in the document]
Comment (MarcelMue) [reply resources/aws/subnet.go]: Not going to do this now, as I am a bit short on time.

**Pull request merged**